optaplanner-8-benchmarks/optaplanner-perf-benchmark/src/test/java/org/jboss/qa/brms/performance/SolverConfigFromXmlTest.java

```diff
@@ -0,0 +1,66 @@
+import org.jboss.qa.brms.performance.examples.travelingtournament.TravelingTournamentExample;
+import org.jboss.qa.brms.performance.examples.tsp.TravelingSalesmanExample;
+import org.jboss.qa.brms.performance.examples.vehiclerouting.VehicleRoutingExample;
+import org.jboss.qa.brms.performance.realworld.cloudbalancing.CloudBalanceBenchmark;
+import org.jboss.qa.brms.performance.realworld.conferencescheduling.ConferenceSchedulingBenchmark;
+import org.jboss.qa.brms.performance.realworld.nurserostering.NurseRosteringBenchmark;
+import org.jboss.qa.brms.performance.score.constraintstream.FlightCrewSchedulingConstraintStreamsBenchmark;
+import org.jboss.qa.brms.performance.score.constraintstream.TravelingTournamentConstraintStreamsBenchmark;
+import org.jboss.qa.brms.performance.score.constraintstream.VehicleRoutingConstraintStreamsBenchmark;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Timeout;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.optaplanner.core.api.solver.SolverFactory;
+import org.optaplanner.core.config.constructionheuristic.ConstructionHeuristicType;
+import org.optaplanner.core.config.localsearch.LocalSearchPhaseConfig;
+import org.optaplanner.core.config.solver.SolverConfig;
+import org.optaplanner.core.config.solver.termination.TerminationConfig;
+
+class SolverConfigFromXmlTest {
+
+    private static final TerminationConfig oneStep = new TerminationConfig();
+
+    private static Stream<Arguments> problemSetAndSolverConfig() {
+        return Stream.of(
+                Arguments.of(new CloudBalanceBenchmark(CloudBalancingExample.DataSet.CB_100_300), Examples.CLOUD_BALANCING.getSolverConfigFromXml()),
+                Arguments.of(new ConferenceSchedulingBenchmark(ConferenceSchedulingExample.DataSet.TALKS_36_TIMESLOTS_12_ROOMS_5), Examples.CONFERENCE_SCHEDULING.getSolverConfigFromXml()),
+                Arguments.of(new FlightCrewSchedulingConstraintStreamsBenchmark(FlightCrewSchedulingExample.DataSet.EUROPE_175_FLIGHTS_7_DAYS), Examples.FLIGHT_CREW_SCHEDULING.getSolverConfigFromXml()),
+                Arguments.of(new NurseRosteringBenchmark(NurseRosteringExample.DataSet.SPRINT), Examples.NURSE_ROSTERING.getSolverConfigFromXml()),
+                Arguments.of(new TSPConstructionBenchmark(ConstructionHeuristicType.FIRST_FIT, TravelingSalesmanExample.DataSet.LU_980), Examples.TRAVELING_SALESMAN.getSolverConfigFromXml()),
+                Arguments.of(new TravelingTournamentConstraintStreamsBenchmark(TravelingTournamentExample.DataSet.SUPER_06), Examples.TRAVELING_TOURNAMENT.getSolverConfigFromXml()),
+                Arguments.of(new VehicleRoutingConstraintStreamsBenchmark(VehicleRoutingExample.DataSet.VRP_USA_100_10), Examples.VEHICLE_ROUTING.getSolverConfigFromXml())
+        );
+    }
+
+    @BeforeAll
+    public static void setTermination() {
```

Comment:
I don't see why we need to create the termination here; it could be done either in the static field if we want to create just a single instance:
`oneStep = new TerminationConfig().withStepCountLimit(1)`, 
or directly in the test method.